### PR TITLE
Upgraded frontend-build version to v12

### PR DESCRIPTION
**Pull request by BilalQamar95**
**Ticket:**
[42: Upgrade eslint to v8.x](https://github.com/openedx/frontend-wg/issues/42)

**What changed?**
- Updated `frontend-build `to v12 (Eslint was updated in `frontend-build` version resulting in it's version being bumped to v12. This PR updates `frontend-build` to reciprocate eslint version update)
- Resolved `eslint` issues post update

### CODE REVIEW

File: src/generic/LoadingPage.jsx [hunk 1]
```diff
@@ -2,34 +2,38 @@ import React from 'react';
...
-    <div className="spinner-border text-primary text-center align-self-center" role="status">
-      {loadingMessage && (
-      <span className="sr-only">
-        {loadingMessage}
-      </span>
-      )}
-    </div>
-  </Col>
-);
+function LoadingPage({ loadingMessage }) {
```
Comment (brian-smith-tcril) [inline src/generic/LoadingPage.jsx:5]: It seems all of the `const` components have been replaced with `function`

Is there something in the linter that's requiring that?

Comment (BilalQamar95) [reply src/generic/LoadingPage.jsx:5]: Yes, this is due to satisfaction of an automatically fixable `react/function-component-definition` rule. You can find more details on the rule [here](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/function-component-definition.md).

**Review by brian-smith-tcril — APPROVED** [review]
:shipit:

**Comment by arbrandes** [comment]
@BilalQamar95, we'd like to merge this, but there are conflicts with master.  Do you mind squashing/rebasing manually?

**Comment by arbrandes** [comment]
Never mind, we can do Squash and merge, apparently :)

**Pull request merged**